From 9c424031c39d4be84a61b46b1c6026e9964e7ed5 Mon Sep 17 00:00:00 2001
From: Xitij Ritesh Patel <xpatel@pulsecode.ca>
Date: Thu, 31 May 2012 00:22:47 -0400
Subject: [PATCH 1/2] Disabled build of generator and shiboken

Disabled the building of generator and shiboken because we'd rather use our
own. Moreover, this one is just buggered up. It tries to build it for ARM, when
we need it for x86/our host, not our target.
---
 CMakeLists.txt |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8f07372..d2d6839 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -115,7 +115,7 @@ add_subdirectory(libshiboken)
 add_subdirectory(doc)
 
 # deps found, compile the generator.
-if (QT4_FOUND AND PYTHONLIBS_FOUND AND PYTHONINTERP_FOUND)
+if (FALSE AND QT4_FOUND AND PYTHONLIBS_FOUND AND PYTHONINTERP_FOUND)
     add_subdirectory(generator)
     add_subdirectory(shibokenmodule)
 
-- 
1.7.5.4


From f6f39f93aa97ea6bd9e829ae487ff5c201fce870 Mon Sep 17 00:00:00 2001
From: Xitij Ritesh Patel <xpatel@pulsecode.ca>
Date: Thu, 31 May 2012 00:24:53 -0400
Subject: [PATCH 2/2] Removed a newline

Removed a newline from this file. Yeah, I don't know either, don't ask.
---
 shibokenmodule/CMakeLists.txt |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/shibokenmodule/CMakeLists.txt b/shibokenmodule/CMakeLists.txt
index 41b20cd..c3c0891 100644
--- a/shibokenmodule/CMakeLists.txt
+++ b/shibokenmodule/CMakeLists.txt
@@ -31,4 +31,4 @@ target_link_libraries(shibokenmodule
 
 add_dependencies(shibokenmodule shiboken)
 
-install(TARGETS shibokenmodule DESTINATION ${PYTHON_SITE_PACKAGES})
\ No newline at end of file
+install(TARGETS shibokenmodule DESTINATION ${PYTHON_SITE_PACKAGES})
-- 
1.7.5.4

